aws/rapids_sagemaker_hpo.ipynb

```diff
@@ -416,1233 +541,114 @@
-      "s3 model output  =\ts3://sagemaker-us-west-2-561241433344/trained-models\n",
-      "compute          =\tsingleGPU\n",
-      "algorithm        =\tXGBoost, 3 cv-fold\n",
-      "instance         =\tml.p3.2xlarge\n",
-      "spot instances   =\tTrue\n",
-      "hpo strategy     =\tRandom\n",
-      "max_experiments  =\t2\n",
-      "max_parallel     =\t2\n",
-      "max runtime      =\t86400 sec\n"
-     ]
-    }
-   ],
-   "source": [
-    "summarize_choices(s3_data_input, s3_model_output, ml_workflow_choice, algorithm_choice, \n",
-    "                  cv_folds, instance_type, use_spot_instances_flag, search_strategy, \n",
-    "                  max_jobs, max_parallel_jobs, max_duration_of_experiment_seconds)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "<span style=\"display: block; text-align: center; color:#8735fb; font-size:30pt\"> **1. ML Workflow** </span>"
+    "use_spot_instances_flag = False"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "8e01d878",
    "metadata": {},
    "source": [
-    "<img src='img/ml_workflow.png' width='800'> "
+    "<span style=\"color:#8735fb; font-size:20pt\"> 2.2 - Create Estimator </span>"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "631a38c5",
    "metadata": {},
    "source": [
-    "<span style=\"color:#8735fb; font-size:20pt\"> 1.1 - Dataset </span>\n",
-    "<a id ='dataset'></a>"
+    "Having built our container [ +custom logic] and pushed it to ECR, we can finally compile all of efforts into an Estimator instance."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb0845d9",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "The default settings for this demo are built to utilize the Airline dataset (Carrier On-Time Performance 1987-2020, available from the [Bureau of Transportation Statistics](https://transtats.bts.gov/Tables.asp?DB_ID=120&DB_Name=Airline%20On-Time%20Performance%20Data&DB_Short_Name=On-Time#)). Below are some additional details about this dataset, we plan to offer a companion notebook that does a deep dive on the data science behind this dataset. Note that if you are using an alternate dataset (e.g., NYC Taxi or BYOData) these details are not relevant.\n",
-    "\n",
-    "The public dataset contains logs/features about flights in the United States (17 airlines) including:\n",
-    "\n",
-    "* Locations and distance  ( `Origin`, `Dest`, `Distance` )\n",
-    "* Airline / carrier ( `Reporting_Airline` )\n",
-    "* Scheduled departure and arrival times ( `CRSDepTime` and `CRSArrTime` )\n",
-    "* Actual departure and arrival times ( `DpTime` and `ArrTime` )\n",
-    "* Difference between scheduled & actual times ( `ArrDelay` and `DepDelay` )\n",
-    "* Binary encoded version of late, aka our target variable ( `ArrDelay15` )\n",
+    "# 'volume_size' - EBS volume size in GB, default = 30\n",
```

Comment:
Is this the right parameter to modify for changing the default storage? (if we run out of space for our container)